File: uzplayer/src/main/java/com/uiza/sdk/dialog/setting/OnToggleChangeListener.java
package com.uiza.sdk.dialog.setting;

public interface OnToggleChangeListener {
    /**
     * Called when the checked state of a compound button has changed.
     *
     * @param isChecked The new checked state of buttonView.
     * @return is checked
     */
    boolean onCheckedChanged(boolean isChecked);
}

File: uzplayer/src/main/java/com/uiza/sdk/dialog/setting/OnToggleChangeListener.kt
package com.uiza.sdk.dialog.setting

interface OnToggleChangeListener {
    /**
     * Called when the checked state of a compound button has changed.
     *
     * @param isChecked The new checked state of buttonView.
     * @return is checked
     */
    fun onCheckedChanged(isChecked: Boolean): Boolean
}
